benchmarks/TFLite/android-mali.cmake

```diff
@@ -27,7 +27,8 @@ iree_benchmark_suite(
     "android-mali"
 
   MODULES
-    "${DEEPLABV3_FP32_MODULE}"
+    # TODO(#10748): Disabled since the iree-import-tflite fails on this model.
```

Comment:
Maybe we can move this comment to the CMakeLists.txt and keep the target here? It will be an empty variable anyway.

Reply:
I dont think it matters. Given how long CI takes to run, I'd just try to land this beast.

Reply:
We actually need to comment these out as empty string is still considered as a list item and break the rule.